Jenkinsfile

```diff
@@ -11,6 +11,11 @@ try {
       checkout scm
       // Make sure not to delete the folder that Jenkins allocates to store scripts
       sh 'git clean -ffdx -e .????????'
+      // Use https url to work around the permission problem
+      sh 'git config --file=.gitmodules submodule.realm/realm-library/src/main/cpp/object-store.url https://github.com/realm/realm-object-store.git'
```

Comment:
Specific SHA1 is in the PR, but github doesn't show it. You can see it with gitk after pull to the local. This is only to solve the jenkins problem with `git@` protocol